src/pages/policy-details/policy-details.tsx

```diff
@@ -28,35 +32,73 @@ export default function PolicyDetails() {
   if (!policy) {
     return (
       <BaseLayout subtitle={`Policy ${policyId}`}>
-        <h4 className={styles.heading}>Policy {abbrStr(policyId)}</h4>
+        <h4 className={styles.heading}>Policy</h4>
         {status === 'loading' && <p className={globalStyles.secondaryColor}>Loading...</p>}
         {status === 'loaded' && <p>Unable to find your policy with given id.</p>}
       </BaseLayout>
     );
   }
 
+  const policyIpfsHref = `https://ipfs.io/ipfs/${policy.ipfsHash}`;
```

Comment:
We use the same logic (`getIpfsUrl`) in https://github.com/api3dao/api3-dao-dashboard/blob/main/src/pages/claim-details/claim-details.tsx#L49.

It would be nice to extract this to a helper function.